specification/machinelearningservices/AzureAI.Assets/model.tsp

```diff
@@ -20,8 +21,8 @@ model AssetVersion {
   @visibility("read")
   id: string;
 
-  @doc("Update stage to 'Archive' for soft delete. Default is Development, which means the asset is under development.")
-  stage: string = "Development";
+  @doc("Update stage to 'Archive' to archive the asset. Default is Development, which means the asset is under development.")
+  stage?: string = "Development";
```

Comment:
Should it be `@madeOptional` in 2024-05-01? Or it is a bug in 2024-04-01?

Reply:
yes it was a bug